### ISSUE TITLE
manal upgrade by patch unclearly

### ISSUE THREAD

**Issue by llplmlyd** [pingcap/docs-cn#4997]
File: [/tiup/tiup-cluster.md](https://docs.pingcap.com/zh/tidb/stable/tiup-cluster)
this paper does not stress the hot-fix package its form，if the binary file doesn't tar ,the patch will fail.

**Comment by TomShawn** [comment]
@lucklove PTAL, thanks~

**Comment by llplmlyd** [comment]
![image](https://user-images.githubusercontent.com/33628991/100839034-837ea100-34ae-11eb-8c0b-a363cb116edc.png)
here

**Comment by lucklove** [comment]
Please use this:

```
tar -czf tidb.tar.gz /path/to/tidb-server
tiup cluster patch tidb-dc-test tidb.tar.gz -R tidb
```

**Comment by llplmlyd** [comment]
i had test, that is right,  here i am to tell you the document may need some stress

**Comment by TomShawn** [comment]
@llplmlyd Would you help stress out the information and become a documentation contributor?😀

![image](https://user-images.githubusercontent.com/41534398/100844066-461e1180-34b6-11eb-96ae-d868c0270470.png)
![image](https://user-images.githubusercontent.com/41534398/100844216-84b3cc00-34b6-11eb-9572-b12739194f7d.png)

**Comment by lucklove** [comment]
@llplmlyd Thank you, got it. I'm refactoring the document this time.

**Comment by TomShawn** [comment]
@lucklove Could you 

> tiup cluster patch tidb-dc-test tidb.tar.gz -R tidb

Have you included this in https://github.com/pingcap/docs-cn/pull/4868?

**Comment by lucklove** [comment]
> @lucklove Could you
> 
> > tiup cluster patch tidb-dc-test tidb.tar.gz -R tidb
> 
> Have you included this in #4868?

Yes, here: https://github.com/pingcap/docs-cn/pull/4868/files#diff-62e6002557bd665f04751fd5c4a5705ce3ba54b05f2f2617951fa8b431fc0ad2R21

**Comment by TomShawn** [comment]
@llplmlyd Issue addressed.
